src/unstructured_client/_hooks/custom/split_pdf_hook.py

```diff
@@ -270,7 +285,27 @@ def before_request(
         if split_size >= page_count and page_count == len(pdf.pages):
             return request
 
-        pages = pdf_utils.get_pdf_pages(pdf, split_size=split_size, page_start=page_range_start, page_end=page_range_end)
+        pages = self._get_pdf_pages(
+            pdf,
+            operation_id=operation_id,
+            split_size=split_size,
+            page_start=page_range_start,
+            page_end=page_range_end
+        )
+        # force free PDF object memory
+        del pdf
+        logger.info(
```

Comment:
ahh must have left these while rebasing with main - will be removed in next commit